### PR TITLE
Added project "colorful dots"

### PR DESCRIPTION
**Pull request by piixelcat**
Added documention for "colorful dots" to the project folder.

### CODE REVIEW

**Comment by fabianmoronzirfas** [comment]
Looks good already.  Some remarks.  

- What about the image usage rights? This will be publicly available online. You should reference the original author and see if you can use them
- Please loose the I. Write on a more abstract level.  

btw:  

> The title of your project will be inserted at the top as `h1`, in Markdown `#`. Don't add it again.
[Contributing-Projects#markdown-files--syntax](https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#markdown-files--syntax)

File: projects/colorful-dots/index.md [hunk 1]
```diff
@@ -4,18 +4,18 @@ title: Colorful Dots
 authors: [Julia Hilt]
 authors-url: [https://github.com/piixelcat]
 ---
-# Colorful Dots
+## Colorful Dots
```
Comment (fabianmoronzirfas) [inline projects/colorful-dots/index.md]: ähm. I didn't mean to move it one h lower. The Title will be added by the layout. That means there will be a H1 for colorful dots. So you could remove this all together and keep the levels of headlines as they where.

**Comment by fabianmoronzirfas** [comment]
Please. Correct the heading thing and you are done! Nice work. You did an impressive performance in the seminar. 👍

**Comment by fabianmoronzirfas** [comment]
Awesome! 🎉

**Comment by fabianmoronzirfas** [comment]
Your projects are live https://interface.fh-potsdam.de/gestalten-in-code/projects/ please review for any mistakes

**Comment by piixelcat** [comment]
Looks fine, haven't found any mistakes!

**Pull request merged**